frontend/public/components/storage-class-form.tsx

```diff
@@ -941,28 +937,26 @@ export class StorageClassForm_ extends React.Component<
 
     return (
       <>
-        {dynamicContent}
-
-        {this.storageTypes[this.state.newStorageClass.type].documentationLink && (
-          <div className="form-group">
-            <label>Additional Parameters</label>
-            <p>
-              Specific fields for the selected provisioner. &nbsp;
-              <ExternalLink
-                href={this.storageTypes[this.state.newStorageClass.type].documentationLink}
-                text="What should I enter here?"
-              />
-            </p>
-            <NameValueEditorComponent
-              nameValuePairs={this.state.customParams}
-              nameString="Parameter"
-              nameParameter="parameter"
-              valueString="Value"
-              addString="Add Parameter"
-              updateParentData={this.updateCustomParams}
+        {!_.isEmpty(parameters) && dynamicContent}
+
+        <div className="form-group">
+          <label>Additional Parameters</label>
+          <p>
+            Specific fields for the selected provisioner. &nbsp;
+            <ExternalLink
+              href={this.storageTypes[this.state.newStorageClass.type].documentationLink}
```

Comment:
no external link will be rendered if `documentationLink` is empty, is this expected?

Reply:
for example selecting `block.csi.ibm.com` provisioner will return  
```
<a class="co-external-link" target="_blank" rel="noopener noreferrer">What should I enter here?</a>
```

Reply:
@yapei Made a fix for this. PTAL!

Reply:
@yapei Someone else has just fixed it!

Reply:
this time, `Additional parameters` section will NOT be shown correctly

<img width="680" alt="Screen Shot 2021-03-24 at 7 03 35 PM" src="https://user-images.githubusercontent.com/12692381/112300321-adfd9f80-8cd3-11eb-82f5-dcda06e87bef.png">

Reply:
@yapei Fixed! Thanks!